src/riak_core_stat_calc_proc.erl

```diff
@@ -180,3 +180,8 @@ maybe_tag_stale(Value) ->
             iolist_to_binary(V)
     end.
 
+now_epoch() ->
+    now_epoch(os:timestamp()).
```

Comment:
```suggestion
    os:system_time(second).
```
Or is that now in line with the "now" philosophy?

Reply:
Wasn't aware of os:system_time/1.  Would allow me to simplify a lot of code that does SW = os:timestamp() then time:now_diff(os:timestamp(), SW) div 1000 etc.

I'll change this, and probably gradually change elsewhere.